pipeline/tasks/veneer_tasks.py

```diff
@@ -55,10 +55,10 @@ def _move_to(self, intermediate_config_location, veneer_api_yaml):
                         os.path.abspath(intermediate_config_location),
                         [os.path.abspath(c_out) for c_out in veneer_api_yaml])
 
-        if len(veneer_api_yaml) == 0:
+        if len(veneer_api_yaml) > 1:
+            raise ValueError(error_fmt + 'Multiple locations specified')
+        elif len(veneer_api_yaml) == 0:
             print error_fmt + 'No location specified'
```

Comment:
I think that should be ok. If we decide to add 3-way merge support later, then we can avoid that kind of failure.